test/testdata/lsp/code_actions/extract_variable/invalid.rb

```diff
@@ -0,0 +1,100 @@
+# typed: true
+# selective-apply-code-action: refactor.extract
+# assert-no-code-action: refactor.extract
+def example(x)
+#           ^ apply-code-action: [A] Extract Variable
```

Comment:
I wasn't aware it would work like this, where every code action can use `[A]`. Does it just mean that the test harness should read the contents of the `.A.rbedited` file to verify the resulting contents? It's not trying to like batch them all up and send them all at once, is it?